src/main/java/com/xebialabs/restito/semantics/Action.java

```diff
@@ -293,6 +295,33 @@ public Response apply(Response input) {
     }
 
     /**
+     * Creates a sequence action which contains all passed actions and
+     * executes one by one of them in the same order if {@link Action#apply(Response)} is repeated.
+     * If all passed actions has been already applied it behaves like {@link #noop()} action.
```

Comment:
Don't you think it should be configurable? I.e. Noop, or the first one, or a custom response like 404. It seems like noop can mask problems when all the responses are exhausted and the requests are still coming for some reason.